aep/general/0122/aep.yaml

```
@@ -0,0 +1,9 @@
+---
+id: 122
+slug: resource-paths
+state: approved
+created: 2023-10-11
+placement:
+  category: resource-design
```

```suggestion
  category: resources
```

I renamed all of them to this, although we change it later if we want to. I felt the "design" part was a little redundant.